### ISSUE TITLE
Determine remaining `COp`s for Numba conversion

### ISSUE THREAD

**Issue by brandonwillard** [pymc-devs/aesara#423]
Before we can replace the C backend, we need to make sure all the existing C implementations for `Op`s have Numba equivalents.  Let's look through the codebase and list the remaining `Op`s here.

**Comment by ferrine** [comment]
is this list updated anyhow?

**Comment by brandonwillard** [comment]
> is this list updated anyhow?

I've crossed out the `Op`s for which we don't really need conversions.  The `aesara.sparse` sub-package requires an entirely different approach, and it has its own issue, so those probably shouldn't be considered here.
